i18n/en.json

```diff
@@ -0,0 +1,241 @@
+{
+  "common": {
+    "targets_one": "{{count}} target",
+    "targets_other": "{{count}} targets",
+    "action": {
+      "cancel": "Cancel",
+      "ok": "Ok",
+      "skip": "Skip"
+    }
+  },
+  "shared": {
+    "permissions": {
+      "sudoError": {
+        "title": "Your user doesn't have enough privileges to proceed",
+        "description": "This application requires sudo privileges to be able to write to drives"
+      },
+      "processDiedError": {
+        "title": "The elevated process died unexpectedly",
+        "description": "The process error code was {{code}}"
+      },
+      "polkitError": {
+        "title": "No polkit authentication agent found",
+        "description": "Please install a polkit authentication agent for your desktop environment of choice to continue"
+      }
+    },
+    "messages": {
+      "common": {
+        "target_one": "target",
+        "target_other": "targets",
+        "describeTarget_one": "{{desc}} ({{name}})",
+        "describeTarget_other": "{{count}} targets"
+      },
+      "progress": {
+        "success_other": "Successful $t(messages.common.target, {\"count\": {{count}}})",
+        "failure_other": "Failed $t(messages.common.target, {\"count\": {{count}}})"
+      },
+      "info": {
+        "singleTarget": "to {{description}} ({{displayName}})",
+        "successfulTargets": "to {{successful}} $t(messages.common.target, {\"count\": {{successful}}})",
+        "failedTargets": "and failed to be flashed to {{failed}} $t(messages.common.target, {\"count\": {{failed}}})",
+        "header": "{{imageBasename}} was successfully flashed {{targets}}"
+      },
+      "compatibility": {
+        "sizeNotRecommended": "Not recommended",
+        "tooSmall": "Too small",
+        "locked": "Locked",
+        "system": "System drive",
+        "containsImage": "Source drive",
+        "largeDrive": "Large drive"
+      },
+      "warning": {
+        "tooSmall": "The selected source is {{bytes}} larger than this drive.",
```

Comment:
Maybe `The selected source image is larger than the selected drive`?
"This" might be a little abstract...

Reply:
This is refering to abitti image being larger than usb drive. Maybe `Abitti is larger than the selected drive` would be more suitable?